src/js/dialogs-manager.js

```diff
@@ -289,7 +289,10 @@
 			}
 
 			if (settings.closeButton) {
-				self.addElement('closeButton', '<div><i class="' + settings.closeButtonClass + '"></i></div>');
+				const tabIndex = ( settings.closeButtonOptions && settings.closebuttonOptions.tabIndex ) ? settings.closebuttonOptions.tabIndex : '0',
+					ariaLabel = ( settings.closeButtonOptions && settings.closebuttonOptions.ariaLabel ) ? settings.closebuttonOptions.ariaLabel : 'Close';
+
+				self.addElement('closeButton', '<div tabindex="' + tabIndex + '" role="button" aria-label="' + ariaLabel + ' (Esc)"><i class="' + settings.closeButtonClass + '"></i></div>');
```

Comment:
1. Make a default `closeButtonOptions` option inside `initSettings` method, and move the `closeButtonClass` property inside as `closeButtonOptions.class`.

2. Add `attributes` as a property of `closeButtonOptions` as an empty object. You can send any attribute to be printed on the button.

3. Then, you won't need to check if `closeButtonOptions` is exists.